src/stactools/hls/fragments/__init__.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import Any, Dict, List, Tuple
+
+import pkg_resources
+from pystac import Extent, Link, MediaType, Provider
+from pystac.utils import make_absolute_href
+
+from stactools.hls.constants import BANDS
+from stactools.hls.utils import band_name_from_href, product_from_href
+
+
+class STACFragments:
+    """Class for accessing collection and asset data."""
+
+    def __init__(self) -> None:
+        self.assets = self._load("assets.json")
+        for key in self.assets.keys():
+            self.assets[key]["type"] = MediaType.COG
```

Comment:
Is `assets` meant to be accessed directly, or only through the `self.asset()` method? If it is not supposed to be used directly, this attribute should be made private (`self._assets`).